api/clients/v2/verification/commitment_utils.go

```diff
@@ -39,11 +38,11 @@ func GenerateBlobCommitment(
 // GenerateAndCompareBlobCommitment generates the kzg-bn254 commitment of the blob, and compares it with a claimed
 // commitment. An error is returned if there is a problem generating the commitment, or if the comparison fails.
 func GenerateAndCompareBlobCommitment(
-	kzgVerifier *verifier.Verifier,
+	g1Srs []bn254.G1Affine,
 	claimedCommitment *encoding.G1Commitment,
 	blobBytes []byte) error {
 
```

Comment:
nit: flip claimedCommitment and blobBytes

Reply:
[c9bdbbe0](https://github.com/Layr-Labs/eigenda/pull/1063/commits/c9bdbbe04da6207372b61445cdc77efb379e4bc2)

Reply:
[1175562b](https://github.com/Layr-Labs/eigenda/pull/1063/commits/1175562bd2cbb2d2954c07d75aeb03c97828ee74)